website/docs/onboarding/containers/index.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 2
+title: Containers
+description: To run container workloads with Podman Desktop, set up at least one container engine.
+tags: [podman-desktop, onboarding]
+keywords: [podman desktop, containers, podman, onboarding]
+---
+
+# Onboarding for container workloads
+
+To run container workloads, set up at least one container engine.
+
+Podman Desktop does not automatically set up container engine resources that you might not need.
+
+#### Procedure
+
+1. Select a container engine supporting your workload.
+
+   | Workload            | Podman | Docker |
+   | :------------------ | :----: | :----: |
+   | Rootless containers | ✅ yes | ❌ no  |
+   | Rootful containers  | ✅ yes | ✅ yes |
+   | Compose             | ✅ yes | ✅ yes |
+   | Pods                | ✅ yes | ❌ no  |
+
+   Podman supports rootless container and pods, in addition to rootful containers and Compose.
+
+2. Select an execution environment supporting your container engine and your operating system.
+
+   - Select a Podman execution environment:
+
+     | Host operating system | Native containers | Podman machine | Lima machine |
```

Comment:
Maybe say "Podman Desktop" here in the header, to explain why `podman machine` is not supported on Linux

Reply:
Does the "disabled in Podman Desktop" mention work for you?

Reply:
Sure, even it might make the table less attractive. I have not yet determined if this is set in stone or not ?

i.e. initially Docker Desktop did not support Linux either, but caved in